dotnet-interactive.Profiler/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Profiler.Api;
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.App;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Jupyter;
+
+namespace dotnet_interactive.Profiler
```

Comment:
The root namespace should be `Microsoft.DotNet.Interactive.App`.

Reply:
done